frontend/src/app/assess/[projectId]/[domain]/[practice]/page.tsx

```
@@ -164,28 +164,28 @@ export default function AssessmentPage() {
           initial={{ opacity: 0, y: 30 }}
           animate={{ opacity: 1, y: 0 }}
           transition={{ duration: 0.8 }}
-          className="glass-effect rounded-2xl p-6 mb-8"
+          className="bg-card rounded-2xl p-6 mb-8 border border-border shadow-sm"
         >
           <div className="flex items-center gap-3 mb-4">
             <button
               onClick={() => router.back()}
-              className="flex items-center gap-2 text-purple-400 hover:text-purple-300 transition-colors"
+              className="flex items-center gap-2 text-primary hover:text-primary/80 transition-colors"
             >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)